packages/forms-web-app/tests/unit/services/department.service.test.js

```diff
@@ -12,14 +13,69 @@ describe('services/department.service', () => {
     jest.resetAllMocks();
   });
 
+  describe('getRefreshedDepartmentData', () => {
+    it('should  give empty result ', async () => {
+      await getLPAList.mockResolvedValue({ data: [] });
```

Comment:
minor - think you can drop the `await` before any call to `.mockResolvedValue`.

Reply:
Oh. Yes. I think I copied from another example but it does no harm so I think I will leave it for refactoring :-)